GCP/Docker/scripts/bootstrap.prefect.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Get the secret from secret manager
```

Comment:
nit: let's add a comment indicating what are the parameters that these functions expect